guides/15_Background_Filter_Video_Processor.md

```diff
@@ -17,9 +17,10 @@ Background replacement is available as part of the Amazon Chime SDK for JavaScri
 
 ### Browser compatibility
 
-A background filter in the Amazon Chime SDK for JavaScript works in Firefox, Chrome, and Chromium-based browsers (including Electron) on desktop, Android, and Safari 14.1 and later on macOS. 
+A background filter in the Amazon Chime SDK for JavaScript works in Firefox, Chrome, and Chromium-based browsers 
+(including Electron) on desktop, Android, and Safari 14.1 and later. 
 
-There is a known issue with `VideoFrameProcessor` in Safari 15: see [github issue 1059](https://github.com/aws/amazon-chime-sdk-js/issues/1059)
+Note that there is a known issue with `VideoFrameProcessor` in Safari 15: see [github issue 1059](https://github.com/aws/amazon-chime-sdk-js/issues/1059). This has been fixed with Safari 16.
```

Comment:
I'm not familiar with this but I feel like we need to clarify what Safari 16 here, is it iOS, iPadOS or macOS? Or they share the same Safari?

Reply:
This is actually browser version and not iOS.